userguide/content/en/docs/Getting started/_index.md

```diff
@@ -255,6 +262,30 @@ gcs_engine_id = "011737558837375720776:fsdu1nryfng"
 
 To use your own Custom Search Engine, replace the value in the `gcs_engine_id` with the ID of your own search engine.
 
+## Known issues
+
+### MacOS
+
+#### `too many open files` - File Descriptors for "hugo server"
+
+By default, MacOS permits a very small number of open File Descriptors which can cause the following error:
+
+```
+ERROR 2020/04/14 12:37:16 Error: listen tcp 127.0.0.1:1313: socket: too many open files
+```
+
+You can temporarily allow more open files by running the following commands:
+
+Note that you might need to set these limits for each new shell. Also, depending on the number of files that you want to
```

Comment:
I'd move this note to below the commands.

Reply:
I wasnt sure what allow too many File Descriptors would do (im not fully familiar with what these settings really do) and wanted to make sure we said something before they ran the command with my generically large number. I slammed today but take a pass to link more Mac ref info about these settings and rewrite this section so that users can make an informed guess / estimate a reasonable value.